File: Tema2/Ejercicio02/Ejercicio2.py
#coding: latin1

class Ejericicio2:
    
    @staticmethod
    def comprobarNumeros():
        numero1 = 0 # Declaramos las variables
        numero2 = 0
        try: # Controlamos las exepciones
            numero1 = int(input("Introduce 1 numero: ")) # Recogemos los numeros del usuario
            numero2 = int(input("Introducelo de nuevo: "))
            
            if (numero1 == numero2): # Comprobamos si los numeros son iguales
                print("Son iguales")
            else:
                print("No son iguales")
        except:
            print("Un numero, no otra cosa!!")    

if __name__ == '__main__':
    Ejericicio2.comprobarNumeros() # Llamamos a la funcion 'comprobarNumeros()' de la clase Ejercicio2.

File: Tema2/Ejercicio02/Ejercicio2.java

import java.util.Scanner;

public class Ejercicio2 {

    /***
     * En nuestro metodo main comprobaremos si los 2 numeros que nos pasen por
     * pantalla son iguales o no.
     * 
     * @param args
     */
    public static void main(String[] args) {
        int numero1Usuario; // Declaramos las variables
        int numero2Usuario;

        Scanner lector = new Scanner(System.in); // Instanciamos Scanner para poder leer los inputs
        try { // Controlamos excepciones
              // Damos las instrucciones y leemos lo que pase por pantalla
            System.out.println("Escribe dos numeros iguales: ");
            numero1Usuario = lector.nextInt();
            System.out.println("y...");
            numero2Usuario = lector.nextInt();

            if (numero1Usuario == numero2Usuario) { // Hacemos la comprobación del contenido de las dos variables, para
                                                    // saber si son iguales o no.
                System.out.println("Los dos numero son iguales");

            } else
                System.out.println("No son iguales");
            lector.close();
        } catch (Exception error) {
            System.err.println("He dicho dos NUMEROS, el codigo de error es: " + error); // Si introduce algo que no sea
                                                                                         // un numero lanzara una
                                                                                         // excepción que recogeremos y
                                                                                         // lanzaremos este mensaje
        }

    }

}
